packages/node/src/configure/SubqueryProject.ts

```
@@ -110,6 +114,26 @@ export class SubqueryProject implements ISubqueryProject {
   }
 }
 
+async function getFileCacheDir(
+  reader: Reader,
+  projectRoot: string,
+  chainId: string,
+): Promise<string> {
+  if (isTmpDir(projectRoot)) return projectRoot;
+  if (reader instanceof LocalReader) {
+    const tmpDir = path.join(os.tmpdir(), `kyveTmpFileCache_${chainId}`);
+    try {
+      await fs.promises.mkdir(tmpDir);
+    } catch (e) {
```

Error gets swallowed